cluster/store/meta_class.go

```diff
@@ -121,19 +122,38 @@ func (m *metaClass) CopyShardingState() *sharding.State {
 	return &st
 }
 
-func (m *metaClass) AddProperty(p models.Property) error {
+func (m *metaClass) AddProperty(props ...*models.Property) error {
 	m.Lock()
 	defer m.Unlock()
 
 	// update all at once to prevent race condition with concurrent readers
-	src := m.Class.Properties
-	dest := make([]*models.Property, len(src)+1)
-	copy(dest, src)
-	dest[len(src)] = &p
+	src := filterOutDup(m.Class.Properties, props)
```

Comment:
can we change the name of this method to `filterOutDuplicates`? it's more descriptive